cmd/snapd-apparmor/export_test.go

```diff
@@ -26,3 +26,12 @@ var (
 	IsContainerWithInternalPolicy = isContainerWithInternalPolicy
 	LoadAppArmorProfiles          = loadAppArmorProfiles
 )
+
+func MockWSL(fun func()) {
+	old := isWsl
+	isWsl = func() bool {
+		return true
+	}
+	fun()
+	isWsl = old
+}
```

Comment:
This could follow our usual approach for mocks, instead of setting `isWsl` to return true and taking in a function to test something. Usually, we would take a mock to assign to `isWsl` and return a function to restore it. The restorer function is then invoked by the caller when the test is done (usually in a defer statement). It would allow the caller to replace `isWsl` with other values and wouldn't require passing test logic into the mock helper.

```suggestion
func MockIsWSL(f func() bool) func() {
	old := isWsl
	isWsl = f
	return func() {
	  isWsl = old
	}
}
```

Reply:
Makes perfect sense to me